libs/shared/ui/src/lib/components/board/board.tsx

```diff
@@ -0,0 +1,255 @@
+import { motion } from 'framer-motion'
+import {
+  type DragEvent,
+  type PropsWithChildren,
+  type ReactNode,
+  createContext,
+  useContext,
+  useId,
+  useState,
+} from 'react'
+import { twMerge } from '@qovery/shared/util-js'
+
```

Comment:
Could you keep the sources that inspired you?

```suggestion
/**
 * Inspired by https://www.youtube.com/watch?v=O5lZqqy7VQE
 * https://www.hover.dev/components/boards
 * https://codesandbox.io/p/sandbox/fvy4m4
 *
 * NOTE: This component doesn't follow the "compound pattern" as
 * - it requires complex typing
 * - it needs to pass props to child components like drag n drop handles
 * - it is rarely used so no big of a need to be highly configurable
 * However in the future it can be turn into a compound pattern using a react context
 */
```

Reply:
https://github.com/Qovery/console/pull/1692/commits/5ca1fa4abc13c7db838a2c90c2521cd5ab6abc6c